src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -54,59 +55,31 @@ inline void calculateBoundingBoxOfTheScene(ExecutionSpace const &space,
       scene_bounding_box);
 }
 
-template <typename ExecutionSpace, typename Primitives, typename MortonCodes>
-inline std::enable_if_t<
-    std::is_same<Box, typename AccessTraitsHelper<AccessTraits<
-                          Primitives, PrimitivesTag>>::type>::value>
-assignMortonCodesImpl(ExecutionSpace const &space, Primitives const &primitives,
-                      MortonCodes morton_codes, Box const &scene_bounding_box)
+template <typename ExecutionSpace, typename Primitives,
+          typename SpaceFillingCurve, typename Projections>
+inline void projectOntoSpaceFillingCurve(ExecutionSpace const &space,
+                                         Primitives const &primitives,
+                                         SpaceFillingCurve const &curve,
+                                         Box const &scene_bounding_box,
+                                         Projections projections)
```

Comment:
Again, make the names consistent with other places. I would prefer `LinearOrderingIndices` to `Projections`.